src/components/WeeklyReport/MonthlyActivityReport.tsx

```diff
@@ -126,6 +126,7 @@ export const MonthlyActivityReport: FunctionComponent<
           activities={activities}
           reportGenerated={reportGenerated}
           hideWeekOf
+          MAR
```

Comment:
Am I missing something here?  It looks like ReportActivitesByParent is only used by MAR.  Future expansion/reuse of that component?

Reply:
We can leave it be.  Like you said before, lots to clean up, and who knows it could become a reusable component.  Was more checking that I wasn't missing some valuable piece.